.chloggen/7517-behind-ff.yaml

```
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. otlpreceiver)
+component: observavibility
```

```suggestion
component: observability
```

or better yet, for consistency with #7483, 

```suggestion
component: service
```